src/wallet/rpcwallet.cpp

```diff
@@ -698,6 +709,7 @@ static RPCHelpMan getreceivedbyaddress()
                 {
                     {"address", RPCArg::Type::STR, RPCArg::Optional::NO, "The bitcoin address for transactions."},
                     {"minconf", RPCArg::Type::NUM, /* default */ "1", "Only include transactions confirmed at least this many times."},
+                    {"include_immature", RPCArg::Type::BOOL, /* default */ "false", "Include immature coinbase transactions."},
```

Comment:
Should this and the other updated RPCs throw if a user supplies `include_immature=true` and `deprecatedrpc=filter_coinbase` is set. That would be inconsistent.